ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOPipelineModel.scala

```diff
@@ -110,6 +110,39 @@ class H2OMOJOPipelineModel(val mojoData: Array[Byte], override val uid: String)
   override def transformSchema(schema: StructType): StructType = {
     StructType(schema ++ predictionSchema())
   }
+
+  def getInputNames(): Seq[String] = {
+    val inputMeta = getOrCreateModel().getInputMeta
+    (0 until inputMeta.size()).map(idx => inputMeta.getColumnName(idx))
```

Comment:
Btw: we should change MOJO api first to get column list directly.

Reply:
agree, will do